connectors/utils.py

```diff
@@ -677,12 +677,25 @@ def __init__(self):
         ) as file:
             config = yaml.safe_load(file)
 
-        self.host = config["extraction_service"]["host"]
-        self.use_file_pointers = config["extraction_service"]["text_extraction"][
-            "use_file_pointers"
-        ]
         self.session = None
 
+        self.extraction_config = config.get("extraction_service", None)
+        if self.extraction_config is not None:
+            self.host = self.extraction_config.get("host", None)
+        else:
+            self.host = None
+
+        if self.host is None:
+            logger.warning(
+                "Extraction service has been initialised but no extraction service configuration was found. No text will be extracted for this sync."
+            )
+
+    def _check_configured(self):
+        if self.host is not None:
```

Comment:
Should check `enabled` as well?

Reply:
We are removing `enabled` entirely. The presence of the config will determine if it is enabled or not.